pkgs/applications/science/electronics/nxpmicro-mfgtools/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  bzip2,
+  libusb1,
+  libzip,
+  openssl,
+  pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  name    = "nxpmicro-mfgtools-1.3.191";
+  version = "uuu_1.3.191";
+
+  meta = with stdenv.lib; {
+    description = "uuu (Universal Update Utility), mfgtools 3.0";
```

Comment:
> For brevity, don’t repeat the name of package — just describe what it does

Check https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

I think a description of what the program does would be better.

Reply:
I wonder if it's better to get the contents of the long description in this field. Then think about what to add in the long description itself.

Reply:
How about this: 5f5d2ee ?